platform/solr/pom.xml

```diff
@@ -23,9 +23,6 @@
     <artifactId>solr</artifactId>
     <packaging>pom</packaging>
     <name>DDF Solr</name>
-    <properties>
-        <saxon.version>9.5.1-5</saxon.version>
```

Comment:
We might need to keep this old version of saxon specifically for Solr. I think they removed some sort of functionality that our Solr instance was depending on. @pklinef would probably remember better than I would.